### PR TITLE
[Fixes] Linter fixes

### PR DESCRIPTION
**Pull request by eriqua**
# Description

>Thank you for your contribution !

A few linter fixes

| Pipeline |
| - |
| |

# Type of Change

Please delete options that are not relevant.

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Update to documentation

# Checklist

- [ ] I'm sure there are no other open Pull Requests for the same update/change
- [ ] My corresponding pipelines / checks run clean and green without any errors or warnings
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (readme)
- [ ] I did format my code

### CODE REVIEW

File: modules/Microsoft.Web/staticSites/readme.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Static Web Sites `[Microsoft.Web/staticSites]`
+# Web Static Sites `[Microsoft.Web/staticSites]`
 
-This module deploys a Static Web Site.
+This module deploys a web static site.
```
Comment (MrMCake) [inline modules/Microsoft.Web/staticSites/readme.md:3]: Why the rename? Should the description be the meaningful manual part. The provider is indeed Web - but that is the case is it is the 'grouping'. However, when talking about a static site, I'd not refer to it as a 'web static site', but definitely 'Static Website', 'Static Web Site' or just 'Static Site', wouldn't you agree?

Comment (MrMCake) [reply modules/Microsoft.Web/staticSites/readme.md:3]: Also we should have this conversation about casing :D I don't know why we change it here as we have both upper & lower casing for Azure services all over the repository. If we decide for one thing or the other we have to go in and update all others.

Comment (eriqua) [reply modules/Microsoft.Web/staticSites/readme.md:3]: Honestly enough the rename is a trick for the linter check. However, I didn't find a specific reference to a static web site in the docs, that's why I thought the workaround would have been acceptable.

On the casing I agree, let's have the discussion first if that is useful. I'm just trying to align to what I see most commonly used throughout the repo when I face it.

Comment (eriqua) [reply modules/Microsoft.Web/staticSites/readme.md]: Agreed and updated

**Review by MrMCake — APPROVED** [review]

**Pull request merged**